unittests/AST/ASTImporterTest.cpp

```diff
@@ -2717,27 +2728,33 @@ TEST_P(ImportFunctionsVisibility, ImportAfterImport) {
     EXPECT_FALSE(ToF1->getPreviousDecl());
 }
 
+bool ExpectLink = true;
```

Comment:
I wanted not more "complications" with this, the enum should be inside the fixture class and names of the members would be too long in the initializer list, and in this very simple case it is not necessary. If a new (third) value is used it can be useful.